packages/autorest.python/autorest/codegen/serializers/builder_serializer.py

```diff
@@ -724,10 +726,12 @@ def _initialize_overloads(self, builder: OperationType) -> List[str]:
                 0
             ].parameters.body_parameter.default_content_type
             retval.append(f'content_type = content_type or "{default_content_type}"')
+        initiated = set()
```

Comment:
what is this code doing with initiated?

Reply:
it's for avoiding declare variable with same name for multiple times to fix like this: https://github.com/Azure/autorest.python/pull/1345/files/7d36750cf213ef95cfbe2fd547323637a9d68750#diff-78b5526b488e5c884878de8c63ffb90f2cf6b2ea30119eb011138bcb5d432ce9
Since we are now using `_content` to pass the serialized body, we have this problem for many cases.

I refactored this part to make this logic shorter in code.